pupil_src/shared_modules/imu_timeline.py

```diff
@@ -204,7 +204,7 @@ def update(self, accel, gyro):  # 2-tuples (x, y, z) for accel, gyro
 
         # These are modified to account for Invisible IMU coordinate system and positioning of
         # the IMU within the invisible headset
-        roll = (
+        roll = -(
```

Comment:
@papr this change was to bring the IMU roll result in line with the headpose tracker, i.e. a negative IMU roll rotation should correspond to a negative headpose roll rotation, and vice versa for positive roll. 
Since the headpose tracker has been around longer, it made sense to change the IMU direction.

Reply:
Actually, Cloud has started exporting orientation based on your code already. I think we need to stay consistent with that. Is there a clear definition for what positive or negative roll means? If yes, we should use a reference for all three components calculating orientation. If not, please use the existing orientation components as reference.

Reply:
The IMU orientation uses the right-hand convention: Positive roll is rightward roll (i.e. right ear lower than left). I have reverted the IMU calc to its original form and modified the headpose estimation to use the same right-hand convention